src/modules/rep.ts

```diff
@@ -130,9 +131,6 @@ export class RepModule extends Module {
 		description: "Reputation: View a user's reputation history",
 	})
 	async getrep(msg: Message, @optional user?: User) {
-		if (!msg.member) {
-			return;
-		}
```

Comment:
Any idea why this check existed?

Reply:
It was just me getting confused with the types😅 We just need to check the sender's ID, but I started with `GuildMember` which is too strict. The check is moot.

Reply:
It would [seem so](https://github.com/typescript-community/community-bot/blame/cbcc92f9475039b180c80a727594d0c7ca65d7ad/src/modules/rep.ts#L129). Cool.